embed-builder/README.md

```diff
@@ -12,26 +10,29 @@ messages.
 
 Build a simple embed:
 
-```rust,no_run
+```rust
 use twilight_embed_builder::{EmbedBuilder, EmbedFieldBuilder};
 
 let embed = EmbedBuilder::new()
     .description("Here's a list of reasons why Twilight is the best pony:")
     .field(EmbedFieldBuilder::new("Wings", "She has wings.").inline())
     .field(EmbedFieldBuilder::new("Horn", "She can do magic, and she's really good at it.").inline())
     .build()?;
+
+# Ok::<(), Box<dyn std::error::Error>>(())
```

Comment:
This is ugly and will render in the readme. We need to remove this.

Reply:
There's no pretty way around it, adding main is ugly too. But `# Ok::<....` will probably immediately be interpreted as something one should ignore.

This PR can stall until embed-builder is part of util (when this won't be a problem).

Reply:
Is this still an issue now that embed-builder is deprecated or should it be an exception, and keep the old behavior?

Reply:
<img width="947" alt="image" src="https://user-images.githubusercontent.com/1760581/157933363-be0e99ea-d2ec-4d48-a0c8-daae6f2a7bb8.png">

Reply:
it didn't seem to render fo rme

Reply:
The issue is that the readme, on GitHub, renders it

Reply:
Decided to keep the old behavior on `embed-builder`